src/Level Bars/main.vala

```diff
@@ -32,9 +32,11 @@ public void main () {
 // such as https://github.com/dropbox/zxcvbn
 
 public void estimate_password_strength () {
-    var level = Math.fmin(Math.ceil(entry.text.length / 2), 6);
+    var level = (int) Math.fmin (Math.ceil ((entry.text.length + 1) / 2), 6);
 
-    switch ((int) level) {
+    label_strength.css_classes = new string[0];
```

Comment:
Why is this line necessary?

Reply:
it resets the css_classes so that the label does not stay the same color as last added class

![image](https://github.com/workbenchdev/demos/assets/128870015/e51be7f0-bc26-4da3-afda-cd508fb18e25)

![image](https://github.com/workbenchdev/demos/assets/128870015/73979959-2308-4f75-a493-1b4da3fb9725)

![image](https://github.com/workbenchdev/demos/assets/128870015/fbd7aeac-8cd5-470d-9d10-fe7d2ae227a7)

these are examples of what happens when you return to a previous label strength without this line of code
It was the main fix that i could find for the issue, however if there is a better way of doing this please let me know

Reply:
Oh I missed this detail, good catch! That is indeed the solution